### PR TITLE
changecase: take input type and metadata from input

### PR DESCRIPTION
**Pull request by bernt-matthias**
fixes https://github.com/galaxyproject/tools-devteam/issues/557

### CODE REVIEW

File: tools/change_case/.shed.yml [hunk 1]
```diff
@@ -6,5 +6,6 @@ long_description: |
   or lowercase.
 name: change_case
 owner: devteam
+homepage_url: ''
```
Comment (nsoranzo) [inline tools/change_case/.shed.yml:9]: If there's no relevant homepage, it's fine to leave this out, `planemo shed_lint` gives just a warning.

```suggestion
```

File: tools/change_case/changeCase.xml [hunk 2]
```diff
@@ -1,77 +1,80 @@
...
-</tool>
+<tool id="ChangeCase" name="Change Case"  version="1.0.0">
+  <description> of selected columns</description>
+  <requirements>
+    <requirement type="package" version="5.22.0">perl</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" level="fatal" />
+  </stdio>
+  <command>perl '$__tool_directory__/changeCase.pl' '${input}' '${cols}' '${delimiter}' '${casing}' '${out_file1}'</command>
```
Comment (nsoranzo) [inline tools/change_case/changeCase.xml:9]:
```suggestion
  <command>perl '$__tool_directory__/changeCase.pl' '${input}' '${cols}' ${delimiter} ${casing} '${out_file1}'</command>
```

File: tools/change_case/changeCase.xml [hunk 3]
```diff
@@ -1,77 +1,80 @@
...
-  windows,is,bad
-
-will result in::
-
-  APPLE is GOOD
-  WINDOWS is BAD
-
-  </help>
-</tool>
+<tool id="ChangeCase" name="Change Case"  version="1.0.0">
```
Comment (nsoranzo) [inline tools/change_case/changeCase.xml:1]: Need to increase the tool version.

File: tools/change_case/changeCase.xml [hunk 4]
```diff
@@ -1,77 +1,80 @@
...
+  <description> of selected columns</description>
+  <requirements>
+    <requirement type="package" version="5.22.0">perl</requirement>
```
Comment (nsoranzo) [inline tools/change_case/changeCase.xml:4]: I see there is a perl 5.26 on conda-forge, should we go with that?

File: tools/change_case/changeCase.xml (hunk 3 above)
Comment (bernt-matthias) [reply tools/change_case/changeCase.xml]: done

File: tools/change_case/changeCase.xml (hunk 4 above)
Comment (bernt-matthias) [reply tools/change_case/changeCase.xml]: good idea. 5.26 is also used in IUC tool requirements

**Comment by nsoranzo** [comment]
Thanks @bernt-matthias !

**Pull request merged**